src/libraries/Microsoft.Extensions.HostFactoryResolver/tests/CreateHostBuilderInvalidSignature/Program.cs

```diff
@@ -2,14 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using MockHostTypes;
+using Microsoft.Extensions.Hosting;
```

Comment:
Are the changes to this file and `tests/CreateHostBuilderPatternTestSite/Program.cs` necessary?

Reply:
Because I added Hosting as a dependency yes.